MdeModulePkg/Core/Dxe/DxeMain.inf

```diff
@@ -95,6 +95,7 @@
   CpuExceptionHandlerLib
   PcdLib
   ImagePropertiesRecordLib
+  MemoryBinOverrideLib      ## MU_CHANGE

```

Comment:
@cfernald, is this library still useful? If not, perhaps we don't bring to 2405. If it is, then let's.

Reply:
After talking to @cfernald we believe that this library is no longer necessary, and I'll be closing this PR.